src/components/widgets/timelapse/TimelapseRenderSettingsDialog.vue

```diff
@@ -0,0 +1,347 @@
+<template>
+  <v-dialog
+    :value="open"
+    :max-width="640"
+    @click:outside="$emit('close')"
+  >
+    <v-card v-if="open">
+      <v-card-title class="card-heading py-2">
+        <v-col cols="11">
+          <v-icon left>
+            $cog
+          </v-icon>
+          <span class="focus--text">
+            {{ $t('app.timelapse.title.render_settings') }}
+          </span>
+        </v-col>
+
+        <v-col
+          class="text-right"
+          cols="1"
+        >
+          <v-icon @click="$emit('close')">
+            $close
+          </v-icon>
+        </v-col>
```

Comment:
```suggestion
        <span class="focus--text">{{ $t('app.timelapse.title.render_settings') }}</span>

        <v-spacer />
        <app-btn
          color=""
          icon
          @click="$emit('close')"
        >
          <v-icon>$close</v-icon>
        </app-btn>
```

This should make it match the look of other dialog windows (like "keyboard shortcuts" or "version information")

![image](https://user-images.githubusercontent.com/85504/163144727-0ef81037-5079-4380-b6ef-37a731217d32.png)